File: TP2DPBO2025C2/Python/Kode Program/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
     
    def __init__(self,ID, Nama, harga, stok, jenis, bahan, warna, untuk, size, merk):
        super().__init__(ID, Nama, harga, stok, jenis, bahan, warna)
        self.untuk = untuk
        self.size = size
        self.merk = merk

    def setUntuk(self, untuk):
        self.untuk = untuk

    def setSize(self, size):
        self.size = size

    def setMerk(self, merk):
        self.merk = merk

    def getUntuk(self):
        return self.untuk

    def getSize(self):
        return self.size

    def getMerk(self):
        return self.merk
    
    def __str__(self):
        return super().__str__()

File: TP2DPBO2025C2/Java/Kode Program/Baju.java
public class Baju extends Aksesoris{
    
    private String untuk;
    private String size;
    private String merk;

    Baju(){

    }

    Baju(String untuk, String size, String merk){
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    Baju(String id, String nama, String harga, int stok, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id, nama, harga, stok, jenis, bahan, warna); // Memanggil konstruktor superclass (Aksesoris)
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public void set_untuk(String untuk){
        this.untuk = untuk;
    }

    public void set_size(String size){
        this.size = size;
    }

    public void set_merk(String merk){
        this.merk = merk;
    }


    public String getUntuk() {
        return untuk;
    }

    public String getMerk() {
        return merk;
    }

    public String getSize() {
        return size;
    }

}
